PubNub/Core/PubNub+Core.m

```diff
@@ -16,6 +16,9 @@
 #import "PNNetwork.h"
 #import "PNHelpers.h"
 
+#ifdef PGDROID
+#import <UIKit/UIKit.h>
```

Comment:
Fix a few more compiler errors : 
```
/Users/aaron/A/dragons3d/Externals/pgengine/Externals/PubNub/PubNub/Core/PubNub+Core.m:219:41: error: use of undeclared identifier 'UIApplicationWillEnterForegroundNotification'
                                   name:UIApplicationWillEnterForegroundNotification object:nil];
                                        ^
/Users/aaron/A/dragons3d/Externals/pgengine/Externals/PubNub/PubNub/Core/PubNub+Core.m:221:41: error: use of undeclared identifier 'UIApplicationDidEnterBackgroundNotification'
                                   name:UIApplicationDidEnterBackgroundNotification object:nil];
                                        ^
```

It's likely that something should be including UIKit/UIKit.h by default on Apportable but isn't for some reason.